libs/shared/ui/src/lib/components/buttons/button/button.tsx

```diff
@@ -1,4 +1,4 @@
-import { Link } from 'react-router-dom'
+import { Link as LinkRouter } from 'react-router-dom'
```

Comment:
Aaah you mean that before this fix, it was calling itself in loop? Or? Why did you have to rename it that way?